e2e/tests/transfer/authz_test.go

```diff
@@ -22,34 +21,31 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
 )
 
-func TestAuthzTransferTestSuite(t *testing.T) {
-	testifysuite.Run(t, new(AuthzTransferTestSuite))
-}
-
-type AuthzTransferTestSuite struct {
-	testsuite.E2ETestSuite
-}
+func (s *TransferTestSuite) TestAuthz_MsgTransfer_Succeeds() {
+	t := s.T()
 
-func (suite *AuthzTransferTestSuite) TestAuthz_MsgTransfer_Succeeds() {
-	t := suite.T()
 	ctx := context.TODO()
 
-	relayer, channelA := suite.SetupChainsRelayerAndChannel(ctx, suite.TransferChannelOptions())
-	chainA, chainB := suite.GetChains()
+	chainA, chainB := s.GetChains()
+	relayer := s.GetRelayerFromSuite()
+
+	channelA, err := relayer.GetChannels(ctx, s.GetRelayerExecReporter(), chainA.Config().ChainID)
+	s.Require().NoError(err)
+	chainAChannels := channelA[len(channelA)-1]
 
 	chainADenom := chainA.Config().Denom
 
-	granterWallet := suite.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
+	granterWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
```

Comment:
this PR is already quite large, can we avoid any cosmetic changes for now? Thank you!